common/src/commonMain/kotlin/io/xorum/codeforceswatcher/features/users/UsersRepository.kt

```diff
@@ -0,0 +1,17 @@
+package io.xorum.codeforceswatcher.features.users
+
+import io.ktor.client.request.*
+import io.xorum.codeforceswatcher.network.responses.backend.HttpClientFactory
+import io.xorum.codeforceswatcher.network.responses.backend.UserData
+import io.xorum.codeforceswatcher.network.responses.backend.request
+
+internal class UsersRepository(token: String?) {
+
+    private val ktorResponseClient = HttpClientFactory().create(token)
+
+    suspend fun fetchUsersData(handles: String) = request {
+        ktorResponseClient.get<UserData>("user/fetch-user-data") {
```

Comment:
Naming of this route looks a bit strange for a few reasons:
- `fetch` is already implied because it's a `get` route
- `user` is already there as well in `user/` prefix

So it should be rather `user/data`